app/lib/routes/home.dart

```diff
@@ -22,7 +23,7 @@ class _HomeState extends State<Home> {
 
   final List<Bike> bikes;
   int selectedBikeIdx = 0;
-  bool useDummy = true;
+  bool useDummy = kDebugMode;
```

Comment:
I think we should move this to the settings, so you can toggle debug bike there but for now this is ok

Reply:
Good idea 👍